README.rst

```diff
@@ -309,7 +309,75 @@ reason is that using the inject method the network devices are not initialized
 in time of parsing kickstart and obtaining information from sysfs (mostly
 getting hw address) fails which results in incomplete ifcfg file generated.
 
+Chapter 7. Continuous Integration structure
+===========================================
+
+Regular test runs
+-----------------
+Every night, the `scenarios workflow`_ runs all tests on all our supported
+operating systems/repositories, like "Fedora Rawhide" or "RHEL 8". These are
+defined in the `containers/runner/scenario`_ script, which essentially calls
+the runner container's ``launch`` script documented above with the desired
+parameters.
+
+The ``rawhide`` and ``daily-iso`` scenarios can in principle run on any host
+that has enough resources. The ``rhel8`` test however needs to run on RHEL
+internal infrastructure.
+
+Currently all scenarios run on `self-hosted GitHub action runners`_, which are
+running in our upshift cluster. See our internal ``builders.git`` repository
+for details and the launch/setup playbooks. These have little magic, though,
+they mostly just create an OpenStack instance and install/configure the action
+runner binary as a service. All the actual test logic is contained in the
+workflow files and the runner container.
+
+The results can be viewed on the `GitHub Daily run workflows page`_. Each run
+has an artifact attached with the detailed log files. This is currently not
+very comfortable, and we are actively looking for a better solution how to
+publish the test result history.
+
+These tests are expected to succeed normally. On failures, rhinstaller
+maintainers get a "failed workflow" notification email and should investigate
+the cause.
```

Comment:
Should we maybe mention our flakes handling here ? At least give some pointer ? Or is it too much of detail / configuration for this place ?

Reply:
Good point, thanks! Updated accordingly, PTAL?

Reply:
Hm, I can't find the update.

Reply:
Bah, git pilot error, sorry. [for reals now!](https://github.com/rhinstaller/kickstart-tests/compare/3618af2107d50088cb77488e87f1d029925a21c8..c77863fcdf02db17541956819ae114ac2ab18165)